core/src/main/java/io/aiven/klaw/controller/TemplateMapController.java

```diff
@@ -48,7 +48,7 @@ public String dashboard(
 
   @RequestMapping(value = "/login", method = RequestMethod.GET)
   public String login(ModelMap model, HttpServletRequest request, HttpServletResponse response) {
-    if (ssoEnabled.equals("true")) {
+    if (ssoEnabled.equals("true") || authenticationType.equals("azuread")) {
```

Comment:
`authencticationType` is sometimes compared first and sometimes last (https://github.com/aiven/klaw/pull/228/files#diff-b7b59e6627b36df927dcd982998e05ac8050806617074702668f0ae489a07c29R429 in here the string constant is put first).
This value is in the config, if I'm not mistaken so it probably can't be null. Otherwise I'd stick to the reverse comparison.

For the future, think about converting this to an Enum, might help.

Reply:
Authentication type is now an Enum and reversed the condition in all applicable places.